### PR TITLE
Create a social theme

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Updated content for Social theme

### How to review 
Compare against slides

### Checklist

* [x] New static content marked up for translation
* [ ] Newly defined schema content included in eq-translations repo

### CODE REVIEW

**Review by bitdivision — APPROVED** [review]
Matches the slides.

File: app/themes/social/templates/static/contact-us.html [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                            {{ _("Friday 9am - 8pm") }}<br />
+                                            {{ _("Saturday 9am - 1pm") }}
+                                        </p>
+                                    </div>
+                                    <p>{{ _("This office is closed during public holidays") }}</p>
+                                    <h2 class="question__title neptune">
+                                        {{ _("Email") }}
+                                    </h2>
+                                    <div class="question__description mars">
+                                        <p>{{helpers.email_address(email="surveyfeedback@ons.giv.uk")}}</p>
```
Comment (insacuri) [inline app/themes/social/templates/static/contact-us.html]: :%s/ons.giv.uk/ons.gov.uk/

**Review by insacuri — CHANGES_REQUESTED** [review]

File: app/themes/social/templates/thank-you.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+<div class="u-mb-s">
+    <p>{{ _("Your information has been submitted. You can now close this page") }}.</p>
+</div>
+
+{% block view_submission %}
+{% if is_view_submitted_response_enabled %}
+    <div class="u-mb-s">
+    {% if view_submission_url %}
+        <p class="u-mb-m@s mars" data-qa="view-submission"><a href={{view_submission_url}}>{{ _("View and print a copy of your answers") }}</a></p>
+        <p class="u-mb-s mars">{{ _("Your answers will only be available for <b>%(duration)s</b> to keep them secure.",
```
Comment (dcdarrell9) [inline app/themes/social/templates/thank-you.html]: Translation tag needs to be after secure?

File: app/themes/social/templates/static/contact-us.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
+{% extends 'layouts/_twocol.html' %}
+{% import 'macros/helpers.html' as helpers %}
+
+{% set survey_title  = _("ONS Business Surveys") %}
```
Comment (MebinAbraham) [inline app/themes/social/templates/static/contact-us.html]: This content doesn't suit.

**Review by MebinAbraham — CHANGES_REQUESTED** [review]

**Review by insacuri — APPROVED** [review]

**Review by MebinAbraham — COMMENTED** [review]
Trello card has cookies and privacy page content attachment?

**Comment by jonnyshaw89** [comment]
@MebinAbraham cookies and privacy is out of scope of this card now

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**